code/game/objects/structures/bonfire.dm

```diff
@@ -121,6 +121,15 @@
 	SIGNAL_HANDLER
 	if(burning & !grill)
 		bonfire_burn()
+	//Not currently burning, let's see if we can ignite it.
+	else if(isliving(entered))
+		var/mob/living/burning_body = entered
+		if(burning_body.on_fire)
+			start_burning()
+			visible_message(span_notice("[entered] runs over [src], starting it's fire!"))
```

Comment:
```suggestion
			visible_message(span_notice("[entered] runs over [src], starting its fire!"))
```

Reply:
sorry for not getting to this quicker, but got them both now.